build/makelib/golang.mk

```diff
@@ -176,7 +176,9 @@ go.vet:
 
 .PHONY: go.fmt
 go.fmt:
-	@$(GOHOST) fmt $(GO_ALL_PACKAGES)
+ifneq ($(shell gofmt -d -e $(GO_SUBDIRS)),)
```

Comment:
I don't think you want to do this in a `$(shell)` since this run every time the makefile is loaded and not part of the go.fmt target.